client/api/filesystem/v1alpha1/api.proto

```diff
@@ -16,6 +16,9 @@ service Filesystem {
     // LinkPath creates a local directory symbolic link between a source path 
     // and target path in the host's filesystem
     rpc LinkPath(LinkPathRequest) returns (LinkPathResponse) {}
+
+    //IsLikelyNotMountPoint checks if a given path is mount or not
```

Comment:
Does Windows have the same problem as linux where we a bind mount on the rootfs can't be detected as a mount point?

This is the reason why in Linux we have IsLikelyNotMountPoint and IsNotMountPoint.

If we don't have the same problem in Windows, then I would just go with an "IsNotMountPoint" call, or maybe even "IsMountPoint" so we don't have a negative in the API

Reply:
Thank you @msau42. Moved the calls to IsMountPoint.